eo-maven-plugin/src/main/java/org/eolang/maven/OyRemote.java

```diff
@@ -81,4 +92,49 @@ public Input get(final String name) throws MalformedURLException {
         return new InputOf(url);
     }
 
+    /**
+     * Hash of head master.
+     * @return SHA of commit
+     * @throws IOException if fails
+     * @checkstyle NonStaticMethodCheck (20 lines)
+     */
+    private String getSha() throws IOException {
+        String sha = "master";
+        final String command = String.format(
+            "%s%s",
+            "curl -o tags.txt https://raw.githubusercontent.com/",
+            "objectionary/home/gh-pages/tags.txt"
+        );
+        try {
+            Runtime.getRuntime().exec(command).waitFor();
+        } catch (final InterruptedException exception) {
+            Logger.info(this, exception.toString());
```

Comment:
@yasamprom this is not how InterruptedException must be handled: https://www.yegor256.com/2015/10/20/interrupted-exception.html